cmd/manager/main.go

```diff
@@ -76,6 +68,24 @@ func main() {
 
 	pflag.Parse()
 
+	// add version subcommand to query the operator version
+	if len(os.Args) > 1 {
+		switch os.Args[1] {
+		case "version":
+			fmt.Printf("%s version: %q, go version: %q\n", os.Args[0], version.Get(),
+				version.GoVersion)
+			os.Exit(0)
+		default:
+			fg := strings.Split(os.Args[1], "=")
```

Comment:
Can't you pass the flags with `space` instead of `=` as well?

Reply:
this handles both cases. We can definitely give flags with spaces, but `=` was a corner case which is handled here.

Reply:
So, if you pass `wmco -a b`, the first argument would come as `-a` which will go to default and won't `fg` be `-a` and the replace will be `--` should also fail right?

Reply:
yes, replacing `--` will fail. in my understanding, because its a shorthand, pflag lookup will return a not nil value. It'll be passed on outside the case and it'll throw an error there. 
here's an example run for your command. 
```
bin version-subcmd  ✅  ./windows-machine-config-operator -a b
unknown shorthand flag: 'a' in -a
Usage of ./windows-machine-config-operator:
      --kubeconfig string                Paths to a kubeconfig. Only required if out-of-cluster.
      --master --kubeconfig              (Deprecated: switch to --kubeconfig) The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.
      --zap-devel                        Enable zap development mode (changes defaults to console encoder, debug log level, disables sampling and stacktrace from 'warning' level)
      --zap-encoder encoder              Zap log encoding ('json' or 'console')
      --zap-level level                  Zap log level (one of 'debug', 'info', 'error' or any integer value > 0) (default info)
      --zap-sample sample                Enable zap log sampling. Sampling will be disabled for integer log levels > 1
      --zap-stacktrace-level level       Set the minimum log level that triggers stacktrace generation (default error)
      --zap-time-encoding timeEncoding   Sets the zap time format ('epoch', 'millis', 'nano', or 'iso8601') (default )
unknown shorthand flag: 'a' in -a

```

Reply:
I was referring to known short hand but it seems in this case you don't have any shorthands. I'll LGTM for now. We can come back to this later.